src/tools/dump-tool/DbDumper.cpp

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "DbDumper.h"
+#include "EngineDumper.h"
+#include "storage/CompactionFilter.h"
+#include "fs/FileUtils.h"
+
+namespace nebula {
+
+using nebula::GraphSpaceID;
+using nebula::kvstore::RocksEngine;
+
+void DbDumper::init(SchemaManager *schemaMngPtr, std::vector<std::string> &&storagePaths) {
+    schemaMngPtr_ = schemaMngPtr;
+    storagePaths_ = std::move(storagePaths);
+}
+
+
+void DbDumper::dump() {
+    for (auto& storagePath : storagePaths_) {
+        auto dbPath = folly::stringPrintf("%s/nebula", storagePath.c_str());
+        auto allSpaces = fs::FileUtils::listAllDirsInDir(dbPath.c_str());
+        for (auto &spaceIdInDir : allSpaces) {
```

Comment:
Could you support a flag to specifies the graph space ? Actually I think it's not necessary to dump all spaces.

Reply:
good issue, I will add it.

Reply:
Then user can use --spaces to dump the space he wanted.